types/component/AnonymousChat/BaseChannelItem.types.ts

```diff
@@ -15,17 +32,19 @@ export interface BaseChannelItemProps {
   isCommentExists?: boolean;
   isMe: boolean;
   isOwnSignedPost?: boolean;
-  message: string;
+  message: any;
   name: string;
   onPress?: () => void;
   picture: string;
   postMaker?: any;
   postNotificationMessageText?: string;
   postNotificationMessageUser?: string;
   postNotificationPicture?: string;
-  showPostNotificationStats?: boolean;
   time: string;
   type: BaseChannelItemTypeProps;
   unreadCount?: number;
   upvote?: number;
+  hasFollowButton?: boolean;
+  handleFollow?: () => void;
+  channelType?: string;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes to the `BaseChannelItemProps` interface seem fine. However, the `message` property type has been changed from `string` to `any`. This could potentially lead to issues if not handled properly in the code where this interface is used. It's generally a good practice to avoid using `any` and instead use more specific types. If the message can be of different types, consider creating a type or an interface that includes all possible structures.
```diff
-  message: string;
+  message: any;
+  postNotificationMessageUser?: string;
+  postNotificationPicture?: string;
+  hasFollowButton?: boolean;
+  handleFollow?: () => void;
+  channelType?: string;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->